test/functional/apps/visualize/index.ts

```diff
@@ -7,115 +7,116 @@
-    describe('new charts library', function () {
+    describe('new charts library visualize ciGroup7', function () {
       this.tags('ciGroup7');
 
-      before(async () => {
-        await kibanaServer.uiSettings.update({
-          'visualization:visualize:legacyChartsLibrary': false,
-        });
-        await browser.refresh();
-      });
-
-      after(async () => {
-        await kibanaServer.uiSettings.update({
-          'visualization:visualize:legacyChartsLibrary': true,
-        });
-        await browser.refresh();
-      });
-
-      // Test replaced vislib chart types
-      loadTestFile(require.resolve('./_area_chart'));
-      loadTestFile(require.resolve('./_line_chart_split_series'));
-      loadTestFile(require.resolve('./_line_chart_split_chart'));
-      loadTestFile(require.resolve('./_point_series_options'));
-      loadTestFile(require.resolve('./_vertical_bar_chart'));
-      loadTestFile(require.resolve('./_vertical_bar_chart_nontimeindex'));
+      before(async () => await update(false));
+
+      after(async () => await update(true));
+
+      [
+        // Test replaced vislib chart types
+        './_line_chart_split_chart',
+        './_point_series_options',
+        './_vertical_bar_chart',
+        './_vertical_bar_chart_nontimeindex',
+        './_line_chart_split_series',
+      ].forEach(load);
     });
 
-    describe('', function () {
+    describe('visualize ciGroup9', function () {
       this.tags('ciGroup9');
 
-      loadTestFile(require.resolve('./_embedding_chart'));
-      loadTestFile(require.resolve('./_area_chart'));
-      loadTestFile(require.resolve('./_data_table'));
-      loadTestFile(require.resolve('./_data_table_nontimeindex'));
-      loadTestFile(require.resolve('./_data_table_notimeindex_filters'));
+      [
+        './_embedding_chart',
+        './_area_chart',
+        './_data_table',
+        './_data_table_nontimeindex',
+        './_data_table_notimeindex_filters',
+      ].forEach(load);
 
       // this check is not needed when the CI doesn't run anymore for the OSS
-      if (!isOss) {
-        loadTestFile(require.resolve('./_chart_types'));
-      }
+      if (!isOss) load('./_chart_types');
     });
 
-    describe('', function () {
+    describe('visualize ciGroup10', function () {
       this.tags('ciGroup10');
 
-      loadTestFile(require.resolve('./_inspector'));
-      loadTestFile(require.resolve('./_experimental_vis'));
-      loadTestFile(require.resolve('./_gauge_chart'));
-      loadTestFile(require.resolve('./_heatmap_chart'));
-      loadTestFile(require.resolve('./input_control_vis'));
-      loadTestFile(require.resolve('./_histogram_request_start'));
-      loadTestFile(require.resolve('./_metric_chart'));
+      [
+        './_inspector',
+        './_experimental_vis',
+        './_gauge_chart',
+        './_heatmap_chart',
+        './input_control_vis',
+        './_histogram_request_start',
+        './_metric_chart',
+      ].forEach(load);
```

Comment:
The thing I don't like about this change, is that now this one index file is different from all the rest in the repo.  It's not a bad change by itself but it's not related to the purpose of this PR.

And it uses a local `load` method which, if we want to do something like this, should be more of a common function shared across all the tests.  Maybe we need a `loadTestFiles()` which just takes an array of test names (in a new PR)?

Reply:
I like that idea

Reply:
On second thought, do many of the tests actually need that functionality?  It seem this visualize folder is different, so changing the whole to match one case seems like a small roi.